src/WebJobs.Script.WebHost/Scale/TableEntityConverter.cs

```diff
@@ -33,108 +33,70 @@ internal static class TableEntityConverter
 
             if (!string.IsNullOrWhiteSpace(etag))
             {
-                entity.ETag = etag;
+                entity.ETag = new ETag(etag);
             }
 
             var jo = JObject.FromObject(o);
             foreach (var prop in jo.Properties())
             {
-                if (TryGetEntityProperty(prop, out EntityProperty entityProperty))
+                if (TryGetEntityProperty(prop, out object value))
                 {
-                    entity.Properties.Add(prop.Name, entityProperty);
+                    entity.Add(prop.Name, value);
                 }
             }
 
             return entity;
         }
 
-        public static object ToObject(Type type, DynamicTableEntity entity)
-        {
-            return ToObject(type, entity.Properties);
-        }
-
-        public static TOutput ToObject<TOutput>(IDictionary<string, EntityProperty> properties)
+        public static TOutput ToObject<TOutput>(IDictionary<string, object> properties)
         {
             return (TOutput)ToObject(typeof(TOutput), properties);
         }
 
-        public static object ToObject(Type type, IDictionary<string, EntityProperty> properties)
+        public static object ToObject(Type type, IDictionary<string, object> properties)
         {
             var jo = new JObject();
             foreach (var pair in properties)
             {
-                ApplyProperty(jo, pair.Key, pair.Value);
```

Comment:
How are you making sure there are no breaking changes in here? Wondering if data that has been written with the old code can be read without issues by this new code. We've had versioning issues in the past with this code.

Reply:
I've been running tests with 3 monitors (storage queue, eventhub and service bus topic) with runtime scaling:

1) Configured them on 4.34
2) Added messages to trigger and scale -> Scale samples aggregated on the Table Storage
3) Platform scaling
4) Stopped and deployed the private site extension to the app
5) Continued executing functions, read scale metrics and scaled back out to continue processing

@mathewc is there an specific extension that you believe I should test (or maybe all the supported)? Anything that failed in the past?